src/style.css

```diff
@@ -1,3 +1,111 @@
 body {
   margin: 0;
+  font-family: "Roboto", "Times New Roman", Times, sans-serif;
+  --active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo-img {
+  margin: 0 0 0 50px;
+}
+
+.nav {
+  justify-content: center;
+
+}
```

Comment:
you cant use `justify-content` if this block is not a flexbox, so please remove it